packages/flutter_tools/test/general.shard/android/gradle_errors_test.dart

```diff
@@ -2,8 +2,6 @@
 // Use of this source code is governed by a BSD-style license that can be
```

Comment:
The chrun on this file is due to the addition of all required arguments to each gradle error handler call. These were previously passed in as null